services_app/src/main/java/org/opendatakit/services/preferences/activities/AppPropertiesActivity.java

```diff
@@ -101,12 +110,16 @@ protected void onCreate(Bundle savedInstanceState) {
     preferenceViewModel.getAdminMode().setValue(mAdminMode);
 
     preferenceViewModel.getAdminMode().observe(this, new Observer<Boolean>() {
+      @SuppressLint("WrongConstant")
```

Comment:
Why are we suppressing a lint error? We should try to fix the error even if it means more code changes.

Reply:
Opps... I didn't modify the codebase, I currently don't know why it was suppressed. I'll have a look and make necessary adjustments in the next PR.